packages/teams-js/test/public/appNotification.spec.ts

```diff
@@ -0,0 +1,215 @@
+import { errorLibraryNotInitialized } from '../../src/internal/constants';
+import { GlobalVars } from '../../src/internal/globalVars';
+import { DOMMessageEvent } from '../../src/internal/interfaces';
+import { appNotification, ErrorCode, FrameContexts } from '../../src/public';
+import { app } from '../../src/public/app';
+import { errorNotSupportedOnPlatform } from '../../src/public/constants';
+import { setUnitializedRuntime } from '../../src/public/runtime';
+import { Utils } from '../utils';
+
+/* eslint-disable */
+/* As part of enabling eslint on test files, we need to disable eslint checking on the specific files with
+   large numbers of errors. Then, over time, we can fix the errors and reenable eslint on a per file basis. */
+
+// Unit Test cases for AppNotifications API
+describe('appNotification', () => {
+  let utils = new Utils();
+  const allowedContexts = [
+    FrameContexts.content,
+    FrameContexts.stage,
+    FrameContexts.sidePanel,
+    FrameContexts.meetingStage,
+  ];
+
+  beforeEach(() => {
+    utils = new Utils();
+    utils.mockWindow.parent = undefined;
+    utils.messages = [];
+    GlobalVars.isFramelessWindow = false;
+  });
+
+  afterEach(() => {
+    app._uninitialize();
+    GlobalVars.isFramelessWindow = false;
+  });
+
+  const displayNotificationParam: appNotification.NotificationDisplayParam = {
+    title: 'New Missed Call',
+    content: 'You just received a missed call from Paul Sayer',
```

Comment:
>Paul Sayer [](http://example.com/codeflow?start=51&length=10)

You can't use names in our source code unless they are on the official approved list of fictitious names. Is that where you got the name from?

Even if you weren't intending to refer to a real person, there can be legal issues from people who actually have that name.

You should ensure you are using a name from the approved list -- or, probably more appropriate here, just don't use a name, e.g., "You just received a missed call."

Reply:
No, I didn't get it from an approved list of fictitious names. Alright I didn't know that could come up as an issue, thanks for the enlightenment.

Alright, I'll take off the name.